### PR TITLE
Added various utility classes

### PR DESCRIPTION
**Pull request by atneik**
'ms-u-textAlign-*' for text alignment:
https://github.com/OfficeDev/Office-UI-Fabric/issues/440
'ms-u-screenReaderOnly' for screen readers:
https://github.com/OfficeDev/Office-UI-Fabric/issues/439
'ms-u-textTruncate' and 'ms-u-noWrap' for truncating text with ellipsis
and for disabling text wrapping respectively:
https://github.com/OfficeDev/Office-UI-Fabric/issues/365

### CODE REVIEW

File: src/sass/_Fabric.Utilities.scss [hunk 1]
```diff
@@ -34,3 +34,38 @@
   @include resetPadding;
   @include resetBoxShadow;
 }
+
+// To set various text alignment styles 
+// @param [variable list] $alignments
+@mixin ms-u-textAlign($alignments...) {
+  @each $align in $alignments {
```
Comment (mikewheaton) [inline src/sass/_Fabric.Utilities.scss]: Very nice!

File: src/sass/_Fabric.Utilities.scss [hunk 2]
```diff
@@ -34,3 +34,38 @@
...
+    .ms-u-textAlign-#{$align} {
+      text-align: $align;
+    }
+  }
+}
+
+// To hide content while still making it readable by screen readers (Accessibility)
+@mixin ms-u-sr-only {
```
Comment (mikewheaton) [inline src/sass/_Fabric.Utilities.scss]: Mixin and class name should be camelCase instead of dashed. E.g. `ms-u-srOnly` or `ms-u-screenReaderOnly`

**Comment by Jahnp** [comment]
Looks great! Once the casing for `ms-u-sr-only` has been switched to camelCase, consider this approved.

**Comment by atneik** [comment]
Pushed the one using camelCase for mixins and classes

**Pull request merged**